src/server/TCPServer.cpp

```diff
@@ -21,11 +21,12 @@ TCPServer::TCPServer(boost::asio::io_context &io_context,
       resolver_(io_context),
       app_(new KVStore::AMOStore()),
       num_servers_(servers.size()),
-      round_(new Round(this)) {
+      expected_(0) {
```

Comment:
nit: More descriptive variable name

Reply:
Also it does not appear to be used

Reply:
Following the pseudo-code variable names. Will be implemented in the future.